src/game/gui/textinput.c

```diff
@@ -20,7 +20,7 @@ typedef struct {
     int ticks;
     int dir;
     surface sur;
-    char *buf;
+    str *text;
```

Comment:
could this be just "str text", no need to malloc ?

Reply:
Changed to just str.